### ISSUE TITLE
Windows 10 Annivesary Update Problems

### ISSUE THREAD

**Issue by SebSeifert** [mfreiholz/ts3video#39]
Hallo,
ich hoffe es ist okay auf Deutsch zu schreiben. Seit dem Windows Update gibt es eine allgemeine Problematik mit Webcams unter Windows 10. Ich habe den Registry Workaround aus dem Artikel befolgt und seitdem kann ich in OpenBroadCaster unter Videoformat MJPEG anstatt YUY2 auswählen was ja durch den Registry eintrag kommt. Unter JPEG ist meine WebCam wieder wunderbar flüssig. Aber in deinem Plugin gibt es so eine einstellmöglichkeit nicht und es scheint das YUY2 genutzt zu werden da die WebCam immernoch sehr stark laggt. Getestet mit einer Logitech C920-C unter 1920x1080.
Hier noch der Artikel: http://www.heise.de/newsticker/meldung/Windows-10-Anniversary-Update-macht-Webcams-unbrauchbar-3301250.html?wt_mc=rss.ho.beitrag.rdf

**Comment by mfreiholz** [comment]
Ja, das Plugin verwendet grundsätzlich die YUY2 bzw. YUV Variante, da dies auch das benötigte Format für den Video-Encoder ist. 

Danke für die Info, ich werde mal testen, wie es sich mit den anderen Ausgangsformaten verhält bzw. um es zuviel CPU Zeit zum umwandeln benötigt.
